frontend/src/views/OrderView.vue

```diff
@@ -243,22 +288,31 @@ onMounted(async () => {
               <router-link :to="`${ROUTES.ORDER.path}/${route.params.id}/edit/invoice/new`">
                 <Button label="Rechnung erstellen"></Button>
               </router-link>
+              <router-link :to="`${ROUTES.ORDER.path}/${route.params.id}/edit/overdue/new`">
+                <Button label="Mahnung erstellen"></Button>
+              </router-link>
             </div>
           </div>
           <!-- TODO: handle no suborder for employees -->
           <section>
-            <TabView>
+            <TabView :active-index="getActiveSubOrderIndex()">
```

Comment:
There is a lot of code around this active index. But what does it do?

Reply:
We have a list of links in our "ubersicht" page for orderSubItems.  We need this index to open correctly selected link and show the one specific subitem. all this code was created for this aim.

Reply:
Makes sense 👌🏻